backend/trojan_dining/views.py

```diff
@@ -41,10 +41,18 @@ def get(self, request):
 
 class GetMenu(APIView):
     def get(self, request):
+        print("here")
         # get the date and if it doesnt exist return ''
         date = request.GET.get('date', '')
+
+        # automatically set date if it isn't there already
+
+        if date == '':
+            date = datetime.datetime.now().strftime("%Y-%m-%d")\
```

Comment:
the second parameter in <dictionary>.get is a default parameter if the key doesn't exist, otherwise it returns None

Can we refactor this into:
```
date = request.GET['date']
if Date is None:
    date = datetime.datetime.now().strftime("%Y-%m-%d")
else:
    try:
            date = datetime.datetime.strptime(date, "%Y-%m-%d")
        except:
            return HttpResponse(status=400)
```

Also does this still work when passing in a date parameter?

Reply:
I updated it to be of this format roughly. It seemed to work fine locally